tests/ImageSharp.Tests/TestUtilities/FlagsHelper.cs

```diff
@@ -0,0 +1,242 @@
+﻿// <copyright file="FlagsHelper.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+namespace ImageSharp.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    /// <summary>
+    /// Helper class for flag manipulation, based on
+    /// <see>
+    ///     <cref>http://www.codeproject.com/KB/dotnet/enum.aspx</cref>
+    /// </see>
+    /// </summary>
+    /// <typeparam name="T">Must be enum type (declared using <c>enum</c> keyword)</typeparam>
+    public class FlagsHelper<T>
+        where T : struct, IConvertible
+    {
+        private static readonly EnumConverter Converter;
+
+        static FlagsHelper()
+        {
+            Type type = typeof(T);
+            string[] names = Enum.GetNames(type);
+            var values = (T[])Enum.GetValues(type);
+
+            Converter = new FlagsEnumConverter(names, values);
+        }
+
+        public static T[] GetSortedValues()
+        {
+            T[] vals = (T[])Enum.GetValues(typeof(T));
+            Array.Sort(vals);
+            return vals;
+        }
+
+        public static T Parse(string value, bool ignoreCase = false, bool parseNumeric = true)
+        {
+            return (T)Enum.ToObject(typeof(T), Converter.ParseInternal(value, ignoreCase, parseNumeric));
+        }
+
+        /// <summary>
+        /// Converts enum value to string
```

Comment:
Not sure I understand why this is needed. If we had the strong enum type we could just do enum.ToString no? 😄